src/blocks/venue/edit.js

```diff
@@ -52,6 +52,13 @@ const Edit = ({ attributes, setAttributes }) => {
 			venuePost?.title.rendered ??
 			__('No venue selected.', 'gatherpress');
 
+		useEffect(() => {
+			setAttributes({
+				venueName: name,
```

Comment:
name should be `venueName: venueName` I think it can just be `venueName` in that case if I'm remembering things right... same with `venueAddress`, yeah?